deps/Makefile

```diff
@@ -163,6 +163,21 @@ DEP_LIBS_STAGED := $(filter-out suitesparse suitesparse-wrapper osxunwind,$(DEP_
 ## Common build target prefixes
 
 default: | $(build_prefix)
+
+# Part of the FreeBSD libgcc_s kludge
```

Comment:
this doesn't look deps specific to me, we should maybe rearrange the patchelf definitions so they're available in the top level makefiles